SmartDeviceLink/public/SDLScreenManager.h

```diff
@@ -252,6 +253,17 @@ If set to `SDLDynamicMenuUpdatesModeForceOff`, menu updates will work the legacy
  */
 - (void)endUpdatesWithCompletionHandler:(nullable SDLScreenManagerUpdateCompletionHandler)handler;
 
+#pragma mark - Change Layout
+
+/// Change the current layout to a new layout and optionally update the layout's night and day color schemes. The values set for the text, graphics, buttons and template title persist between layout changes. To update the text, graphics, buttons and template title at the same time as the template, batch all the updates between `beginUpdates` and `endUpdates`. If the layout update fails while batching, then the updated text, graphics, buttons or template title will also not be updated.
+///
+/// If you are connected on a < v6.0 connection and batching the update, the layout will be updated, then the text and graphics will be updated. If you are connected on a >= v6.0 connection, the layout will be updated at the same time that the text and graphics are updated.
+///
+/// If this update is batched between `beginUpdates` and `endUpdates`, the completion handler here will not be called. Use the completion handler on `endUpdatesWithCompletionHandler:`
+/// @param templateConfiguration The new configuration of the template, including the layout and color scheme.
+/// @param handler A handler that will be called when the layout change finished.
+- (void)changeLayout:(SDLTemplateConfiguration *)templateConfiguration withCompletionHandler:(nullable SDLScreenManagerUpdateCompletionHandler)handler;
```

Comment:
The `endUpdates` and `endUpdatesWithCompletionHandler:` method documentation also need to be updated to include information about updating the template.